shared_model/validators/field_validator.cpp

```diff
@@ -275,26 +263,31 @@ namespace shared_model {
 
     void FieldValidator::validateCreatedTime(
         ReasonsGroupType &reason,
-        const interface::types::TimestampType &timestamp) const {
-      iroha::ts64_t now = time_provider_();
-
-      if (now + future_gap_ < timestamp) {
+        interface::types::TimestampType timestamp,
+        interface::types::TimestampType current_timestamp) const {
```

Comment:
Please add docstrings for both `validateCreatedTime` methods. They are needed here more than elsewhere since the lists of methods arguments is quite similar and unclear at the same time.